src/main/kotlin/no/nav/tilleggsstonader/kontrakter/søknad/laeremidler/AvsnittLæremidler.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.tilleggsstonader.kontrakter.søknad.laeremidler
+
+import no.nav.tilleggsstonader.kontrakter.søknad.EnumFelt
+import no.nav.tilleggsstonader.kontrakter.søknad.JaNei
+
+data class UtdanningAvsnitt(
+    val annenUtdanning: EnumFelt<AnnenUtdanningType>? = null,
+    val mottarUtstyrsstipend: EnumFelt<JaNei>? = null,
+    val harFunksjonsnedsettelse: EnumFelt<JaNei>,
+)
```

Comment:
Var litt usikker på hva som skulle få være null her. 

`annenUtdanning` skal kun vises dersom man ikke har en utdanning som kan hentes eller hvis man trykket "annet". Men henting av utdanning er ikke på plass enda

`mottarUtstyrsstipend` skal kun vises til personer under 21 år, men atm vises den for alle.